docs/vhlo.md

```diff
@@ -0,0 +1,128 @@
+# The VHLO Dialect
+
+## What is the VHLO Dialect?
+
+The VHLO (Versioned StableHLO) Dialect is used for serialization and stability.
+It provides a snapshot of the StableHLO dialect at a given point in time,
+denoted by version number.
```

Comment:
How about "It provides a snapshot of the StableHLO dialect at a given point in time by versioning individual program elements"? The current formulation is a bit ambiguous with respect to what "version number" means - the version of the dialect or the version of the op.